src/components/blog-item.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from 'next/image'
+import Link from 'next/link'
+
+import blogStyles from '../styles/blog.module.css'
+
+interface blogProps {
+  image?: string
+  title?: string
+  summary?: string
+  datePublished?: string
+  author?: string
+}
+
+const blogItem = ({
+  image,
+  title,
+  summary,
+  datePublished,
+  author,
+}: blogProps) => {
+  return (
+    <>
+      <div className={blogStyles.blogItem}>
+        {/* eslint-disable-next-line @next/next/no-img-element */}
+        <img src={image} alt={title} className={blogStyles.blogImage} />
```

Comment:
```suggestion
        <Image src={image} alt={title} />
```

Better to use [`next/image`](https://nextjs.org/docs/api-reference/next/image)

Reply:
@paulcretu this returns a strict error that asks for a string even when props are given a type of string. Seems  to be a NextJs issue with the properties of Next/Image

Reply:
What's the exact error you get?

Reply:
It sees the prop as undefined. before it is passed

![image](https://user-images.githubusercontent.com/22577790/153064159-d1113707-0438-4e8d-9603-3abf411022c1.png)

Reply:
Oh in that case you need to make sure that what you're passing in can't be undefined. In your `BlogProps` interface you're currently making the `image` prop [optional](https://www.typescriptlang.org/docs/handbook/interfaces.html#optional-properties) with the `?`:

```
image?: string
```

Which gives it a type of `string | undefined`. Instead you need to either make it mandatory:

```
image: string
```

or do type narrowing to make sure it's not undefined (the [typescript docs](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#truthiness-narrowing) do a good job explaining it)

Reply:
Thank you! Here I was thinking that is only affected for the time when the image might not be available.
Thank you once again @paulcretu